.github/workflows/no-empty-strings.yml

```diff
@@ -34,3 +34,4 @@ jobs:
           GITHUB_REPOSITORY: ${{ github.repository }}
           GITHUB_PR_NUMBER: ${{ github.event.pull_request.number }}
           GITHUB_BASE_REF: ${{ github.base_ref }}
+          EXCLUDED_FILES:
```

Comment:
What's the point of keeping this variable empty? Can't we simply remove it if it's not used?

Reply:
We can but that will fail the CI since the variable is not set

> The empty string CI [check](https://github.com/rndquu/ts-template/actions/runs/12296957394/job/34317047283) is passed for that commit although the expected behavior for the CI to fail since the `EXCLUDED_FILES` env variable is no longer set.

Reply:
Okay just to confirm you want the CI to pass is no `EXCLUDED_FILES ` is passed correct? since it’s optional.

I’m not sure what you mean by

> were no empty strings in the https://github.com/ubiquity/ts-template/blob/78c59b5c2731340d1e0f22402cd183382a9b7c0a/.github/empty-string-checker.ts file for the CI to pass

Do you mean that if the env variable is set then to check the files passed to it?

Reply:
> Okay just to confirm you want the CI to pass if there’s no EXCLUDED_FILES  passed correct?

Yes

> Do you mean that if the env variable is set then to check the files passed to it?

I don't understand the question. 

Make sure:
1. `EXCLUDED_FILES` is optional
2. There are no empty strings in https://github.com/ubiquity/ts-template/blob/78c59b5c2731340d1e0f22402cd183382a9b7c0a/.github/empty-string-checker.ts